src/components/ToastContainer.vue

```diff
@@ -70,10 +70,12 @@ function clearAll() {
 </template>
 
 <style scoped>
+
 .osim-toast-container {
   position: v-bind(position);
   top: v-bind(top);
   /*height: calc(100vh - 100px);*/
+  pointer-events: all !important; /* override bootstrap .toast-container default value of 'none'*/ 
```

Comment:
I usually prefer to try increasing selector specificity to avoid !important, but because pointer-events: none is surprising, this is fine

Reply:
Likewise. I also find it surprising though I assume there's bootstrap rationale for it, so I opted for `!important` in this case to imply the override use case is very specific at present.